### PR TITLE
Add support for Linux ARM64

### PR DESCRIPTION
**Pull request by martin-g**
Related to https://github.com/DecodeGenetics/graphtyper/issues/110

Updates:
* paw to [latest `dev` + ARM64 fixes](https://github.com/hannespetur/paw/pull/16) 
* parallel-hashmap to 1.35

Added a CI job for Linux ARM64. Results could be seen at my [fork](https://github.com/martin-g/graphtyper/actions/runs/3097821664/jobs/5014953708)

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -13,7 +13,8 @@
 	ignore = dirty
 [submodule "paw"]
 	path = paw
-	url = https://github.com/hannespetur/paw.git
+	url = https://github.com/martin-g/paw.git
```
Comment (martin-g) [inline .gitmodules:16]: This is temporary, until my PR in Paw is merged.

**Review by emiliofernandes — APPROVED** [review]
Thank you, @martin-g !
I was able to build and use Graphtyper on AWS Graviton 3 by using your branch !
I hope this PR get merged soon!

**Comment by martin-g** [comment]
@hannespetur Thanks for merging it!
But there was a TODO here - https://github.com/DecodeGenetics/graphtyper/pull/114#pullrequestreview-1115402510
I will send a new PR that uses latest Paw your repo!

**Comment by martin-g** [comment]
https://github.com/DecodeGenetics/graphtyper/pull/120

**Comment by hannespetur** [comment]
Thanks!

**Pull request merged**